akka-http-core/src/main/java/akka/http/javadsl/model/HttpEntity.java

```diff
@@ -40,48 +40,48 @@
     /**
      * Returns the content-type of this entity
      */
-    public abstract ContentType contentType();
+    ContentType contentType();
 
     /**
      * The empty entity.
      */
-    public static final HttpEntityStrict EMPTY = HttpEntity$.MODULE$.Empty();
+    HttpEntityStrict EMPTY = HttpEntity$.MODULE$.Empty();
```

Comment:
Hah, right – thanks guys. It's been a while since I've put a constant in a Java interface. ;-)